specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2019-04-01-preview/managedservices.json

```diff
@@ -0,0 +1,845 @@
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Registration Operations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RegistrationDefinition": {
+      "properties": {
+        "properties": {
+          "description": "Properties of a registration definition.",
+          "$ref": "#/definitions/RegistrationDefinitionProperties"
+        },
+        "plan": {
+          "description": "Plan details for the managed services.",
+          "$ref": "#/definitions/Plan"
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Fully qualified path of the registration definition."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Type of the resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Name of the registration definition."
+        }
+      },
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "description": "Registration definition."
+    },
+    "RegistrationDefinitionProperties": {
+      "properties": {
+          "description": {
+            "type": "string",
+            "description": "Description of the registration definition."
+          },
+          "authorizations": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/Authorization"
+            },
+            "description": "Authorization tuple containing principal id of the user/security group or service principal and id of the build-in role."
+          },
+          "registrationDefinitionName": {
+            "type": "string",
+            "description": "Name of the registration definition."
+          },
+          "managedByTenantId": {
+            "type": "string",
+            "description": "Id of the managedBy tenant."
+          },
+          "provisioningState": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Current state of the registration definition.",
+            "enum": [
+              "NotSpecified",
+              "Accepted",
+              "Running",
+              "Ready",
+              "Creating",
+              "Created",
+              "Deleting",
+              "Deleted",
+              "Canceled",
+              "Failed",
+              "Succeeded",
+              "Updating"
+            ],
+            "x-ms-enum": {
+              "name": "ProvisioningState",
+              "modelAsString": false
```

Comment:
Is there any reason why you don't want `modelAsString` set to true? In case you add an entry later, it may be breaking for older generated clients.

Reply:
Updated. Thanks for the suggestion.